Extensions/PlatformBehavior/PlatformerObjectRuntimeBehavior.h

```diff
@@ -176,16 +178,17 @@ class GD_EXTENSION_API PlatformerObjectRuntimeBehavior
                              // edge is also at X position 10). This parameter
                              // "ignoreTouchingEdges" will be passed to all
                              // collision handling functions.
-  bool roundCoordinates;   ///< true to round coordinates when trying to move on
-                           ///< X and Y axis.
-  double gravity;          ///< In pixels.seconds^-2
-  double maxFallingSpeed;  ///< In pixels.seconds^-1
-  double acceleration;     ///< In pixels.seconds^-2
-  double deceleration;     ///< In pixels.seconds^-2
-  double maxSpeed;         ///< In pixels.seconds^-1
-  double jumpSpeed;        ///< In pixels.seconds^-1
-  double slopeMaxAngle;    ///< In degrees
-  double slopeClimbingFactor;  ///< Equals to tan(slopeMaxAngle).
+  bool roundCoordinates;     ///< true to round coordinates when trying to move on
+                             ///< X and Y axis.
+  double gravity;             ///< In pixels.seconds^-2
+  double maxFallingSpeed;     ///< In pixels.seconds^-1
+  double ladderClimbingSpeed; ///< In pixels.seconds^-1
+  double acceleration;        ///< In pixels.seconds^-2
+  double deceleration;        ///< In pixels.seconds^-2
+  double maxSpeed;            ///< In pixels.seconds^-1
```

Comment:
Lots of lines are marked as changed while it's not the case. Can you verify why? In particular, verify that your editor is using proper line endings.

Reply:
This should be autoformatted by clang-format, so I would rather cancel any changes here.

Reply:
I don't mind but 
this is how the original changes I made look
![Screenshot from 2020-03-27 10-15-59](https://user-images.githubusercontent.com/48093317/77722888-eca34000-7014-11ea-97b3-27f38c0cd354.png)
This is how it looks if I don't make the changes to all the lines
![Screenshot from 2020-03-27 10-21-43](https://user-images.githubusercontent.com/48093317/77722930-017fd380-7015-11ea-9487-d3a78f660f0b.png)
As u can see it is breaking the consistency and its looking odd, though I won't mind keeping it this way if you like it?